src/pybind/mgr/volumes/module.py

```diff
@@ -12,7 +12,6 @@
 
 goodchars = '[A-Za-z0-9-_.]'
 
-
```

Comment:
No, need to remove this line. It's a standard convention to have to 2 empty lines.

Reply:
restored blank line